docs/admin/access.rst

```diff
@@ -214,7 +203,7 @@ Managing users and groups
 -------------------------
 
 All users and the various groups they are in can be managed using the
-Django admin interface available, which you can get to by appending
+available Django admin-interface, which you can get to by appending
```

Comment:
It has a deprecated note though. Sounds like it will be removed fully. (Which would remove some complexity.)

Reply:
It won't be removed soon, I guarantee... Unless some one (namely najel) will reimplement full-featured group/permission management in the weblate UI, but it will likely cause rewrite of most of the access control chapter from scratch yet another time... It's pretty big task which probably won't bring a lot...